eth_utils/applicators.py

```diff
@@ -96,6 +96,21 @@ def apply_one_of_formatters(formatter_condition_pairs, value):
 
 @to_dict
 def apply_key_map(key_mappings, value):
+    key_conflicts = set(
+        value.keys()
+    ).difference(
+        key_mappings.keys()
+    ).intersection(
+        v
+        for k, v
+        in key_mappings.items()
+        if v in value
```

Comment:
@pipermerriam changing this from `if k in value` as you suggested caught the case where it's unsure if a value has been remapped or just passed through i.e. `formatter = {'a': 'b'}` and `value = {'b': 3}`

Reply:
👍 typo on my part, good catch.